src/laplace.jl

```diff
@@ -10,13 +10,13 @@ mutable struct Laplace <: BaseLaplace
     subset_of_weights::Symbol
     hessian_structure::Symbol
     curvature::Union{Curvature.CurvatureInterface,Nothing}
-    σ::Real
-    μ₀::Real
-    μ::AbstractVector
-    P₀::Union{AbstractMatrix,UniformScaling}
-    H::Union{AbstractArray,Nothing}
-    P::Union{AbstractArray,Nothing}
-    Σ::Union{AbstractArray,Nothing}
+    σ::Real                                                                  # prior noise/uncertainty          
```

Comment:
More precisely the standard deviation in the (Gaussian) prior

Reply:
Fixed.